Java-Ds/Queue.java

```diff
@@ -0,0 +1,75 @@
+class Queue{
+    static int front,rear,capacity,queue[];
+
+    Queue(int c){
+        front=rear=0;
+        capacity=c;
+        queue= new int[capacity];
+    }
+
+
+    void enqueue(int item){
+        if (capacity==rear){
+            System.out.println("Queue is full \n");
+        }
+        else{
+            queue[rear]=item;
+            rear++;
+        }
+    }
+
+    static void dequeue(){
```

Comment:
Don't use static methods. All queue operations are tied to the individual object we create and not the class itself